### PR TITLE
Added Binary_Tree_TopView , Tree_Mirror_Image and Binary_Tree_LeftView 

### PR DESCRIPTION
**Pull request by helios1101**
Folders contain cpp, c, java, python codes.

### CODE REVIEW

**Comment by helios1101** [comment]
This is in reference to issue #475 
- [x] Mirror Image Of Tree
- [x] Top View Of Tree
- [ ]  Left View Of Tree

**Comment by helios1101** [comment]
This is in reference to issue #475 
- [x] Mirror Image Of Tree
- [x] Top View Of Tree
- [x]  Left View Of Tree

File: Binary_Tree_TopView/Binary_Tree_TopView.cpp [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+#include <queue>
+#include <map>
+using namespace std;
+
+// Structure of binary tree
+struct Node
+{
+    Node *left;
+    Node *right;
+    int hd;
```
Comment (gupta2140) [inline Binary_Tree_TopView/Binary_Tree_TopView.cpp]: Can you change it to a more meaningful variable name?

Comment (helios1101) [reply Binary_Tree_TopView/Binary_Tree_TopView.cpp]: Sure!

**Comment by jainaman224** [comment]
@helios1101 Do have a look at changes made by me.

File: Tree_Mirror_Image/Tree_Mirror_Image.c [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    int data;
+    struct Node *left;
+    struct Node *right;
+};
+
+struct Node *newNode(int data)
+
+{
+    struct Node *node = (struct Node *)
+        malloc(sizeof(struct Node));
```
Comment (gupta2140) [inline Tree_Mirror_Image/Tree_Mirror_Image.c]: indentation

File: Tree_Mirror_Image/Tree_Mirror_Image.cpp [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    struct Node *left;
+    struct Node *right;
+};
+
+/* Helper function that allocates a new node with
+the given data and NULL left and right pointers. */
+struct Node *newNode(int data)
+{
+    struct Node *node = (struct Node *)
+        malloc(sizeof(struct Node));
```
Comment (gupta2140) [inline Tree_Mirror_Image/Tree_Mirror_Image.cpp]: indentation

**Comment by helios1101** [comment]
It shows I have dismissed @jainaman224  stale review via aa3b657 ...
But it shows me can't push, and showed some options after which i did ...
$ git pull
Message why this merge is necessary ... and i pressed yes
now i did ..
add , commit and push !

Is there anything i did wrong ?? Or anything i should have not done.

File: Tree_Mirror_Image/Tree_Mirror_Image.c (hunk 2 above)
Comment (helios1101) [reply Tree_Mirror_Image/Tree_Mirror_Image.c]: Done

File: Tree_Mirror_Image/Tree_Mirror_Image.cpp (hunk 3 above)
Comment (helios1101) [reply Tree_Mirror_Image/Tree_Mirror_Image.cpp]: Done

**Comment by jainaman224** [comment]
@helios1101 Just a suggestion break this PR into 2-3 PRs, if you are GSSoC aspirant. As maximum marks that be allocated on one PR is 50.

Don't worry about stale reviews.

**Comment by helios1101** [comment]
@jainaman224 Sure !!
Can you please tell me how to do it or provide the leads!

**Comment by helios1101** [comment]
Or otherwise, can I open a different pull request on each folder and delete this one ??
@jainaman224 @gupta2140

**Comment by jainaman224** [comment]
I am closing this PR. I will review all three PR separately.

**Pull request closed without merging**